src/main/riddl/Venues/Reservations/reservationMessages.riddl

```diff
@@ -0,0 +1,45 @@
+command EstablishReservation is {
+  reservationId: ReservationId briefly "A unique identifier for this Reservation. This must be provided by the calling client.",
+  info: ReservationContext.Info briefly "Details about a Reservation required for it to exist. See ReservationContext.Info for details.",
+  establishingMember: MemberId briefly "The Id of the Member who is issuing the create command.",
+  owner: MemberId? briefly "Used in the case that establishingMember is not the owner"
+} briefly "Creates a Reservation"
+
+event ReservationEstablished is {
+  reservationId: ReservationId,
+  info: ReservationContext.Info,
+  meta: ReservationContext.MetaInfo
+}
+
+command EditReservationInfo is {
```

Comment:
I think this could be ChangeReservation or UpdateReservation
I think we could handle updating all details of a Reservation in a single call like this. For Organization we updated specific fields where there was high frequency (addMembers/removeMembers) or high complexity (UpdateParent, AddContacts, etc.), so I thought it necessary to have an UpdateInfo command that changed a block of Info rather than having a bunch of getters/setters for things that didn't fall in the above categories.